### ISSUE TITLE
Suggestion: Chat filtering

### ISSUE THREAD

**Issue by jannicklorenzen** [rsbmatt/OSBuddy#160]
Add an option to put in keywords that will be filtered out in the game (user sent messages).

If possible, do a smart filter a lá RS3.

**Comment by OmaiGrundles** [comment]
Yeah, User-defined keywords to filter out user messages would be *amazing* for people like me who like to socialize in the GE. Having to put in so many people into ignore lists every time I go there is starting to become quite infuriating.

I'm almost worried about going further than user-defined filters though, If certain filters become default for all osbuddy users, then the chatbots might end up finding creative ways to bypass the filters because of how popular osbuddy is. It might take a lot of effort on osbuddy's part to keep up with the chatbots in that case.

Maybe we could add in the ability to import/export text files that have strings(keywords) to be filtered. then if a group goes and makes a big list of keywords, any osbuddy user could download that list and plug it in.
